01_intro-r.qmd

```diff
@@ -508,32 +575,41 @@ This simplifies code and makes it portable because all relative file paths will
 
-## Stack Overflow
+### Stack Overflow
 
 [Stack Overflow](https://stackoverflow.com/) contains numerous solutions. If a problem is particularly perplexing, it is simple to submit questions. Exercise caution when submitting questions because the Stack Overflow community has strict norms about questions and loose norms about respecting novices.
 
-## RStudio community
+### RStudio community
 
 [RStudio Community](https://community.rstudio.com/) is a new forum for R Users. It has a smaller back catalog than Stack Overflow but users are friendlier than on Stack Overflow.
 
-## CRAN task views
+### CRAN task views
 
 [CRAN task views](https://cran.r-project.org/web/views/) contains thorough introductions to packages and techniques organized by subject matter. The Econometrics, Reproducible Research, and and Social Sciences task views are good starting places.
 
-## Twitter
+### Twitter
 
-Twitter is mostly bad. But the #rstats hashtag and #rstats community are mostly good. They are also generally inclusive and civil. In particular, open sources developers like Hadley Wickham (@hadleywickham), Jenny Bryan (@JennyBryan), and Joe Cheng (@jcheng) are active.
+Twitter is mostly bad. But the #rstats hashtag and #rstats community are mostly good. They are also generally inclusive and civil. In particular, open sources developers like Hadley Wickham (\@hadleywickham), Jenny Bryan (\@JennyBryan), and Joe Cheng (\@jcheng) are active.
 
-## Data Science for Public Policy Slack
+### Data Science for Public Policy Slack
 
 We've created a Slack workspace for this class (which will be shared across both sections) and encourage you to ask questions in Slack. In general, we ask that you try to answer questions on your own using the sources above *before* posting in Slack. Practicing finding and applying the relevant information to answer your questions is an important data science skill! The teaching staff will be checking the Slack to help answer questions in a reasonable time frame and we also encourage you to answer each other's questions - it's a great way to improve your R skills!
 
-Questions on Slack must be asked using [reproducible examples](http://adv-r.had.co.nz/Reproducibility.html). Simply copying-and-pasting questions or answers in the Slack channel is not allowed. If you're unsure how to share a reproducible example without sharing your answers in a public channel, you can DM the teaching staff to be safe. 
\ No newline at end of file
+Questions on Slack must be asked using [reproducible examples](http://adv-r.had.co.nz/Reproducibility.html). Simply copying-and-pasting questions or answers in the Slack channel is not allowed. If you're unsure how to share a reproducible example without sharing your answers in a public channel, you can DM the teaching staff to be safe. 
+
+### ChatGPT:
```

Comment:
I knew you would add this! :)

Reply:
@awunderground haha I am not surprised. Let me know if you think that guidance is appropriate. I think it is reasonable and fairly boiler-plate, but happy to make updates based on your guidance.